### DISCUSSION TITLE
现在New Bing是不是用不了了？

### DISCUSSION THREAD

**Discussion by Zhuangmoumou** [PenUniverse/PenMods#198] [你问我答] [answered]
### 避免发送重复的问题报告

- [X] 我已检查所有打开的 Issues，确保这个问题没有被提出过

### 异常模块

Services (PenMods相关服务，如更新服务)

### 词典笔型号

YDP02x

### PenMods 版本

1.20

### 词典笔系统版本

2.07

### 发生了什么?

现在使用New Bing都会出现Json错误，看起来像Bing的主页，我的代理是使用Cloudflare对www.bing.com和sydney.bing.com进行代理的

**Comment by Redbeanw44602** [comment]
必应对不同风险评级的IP待遇不同，建议你找干净的IP。

**Reply by Zhuangmoumou** [reply]
![ab96ab6e-50a9-4909-9c16-70baa20aa2a8](https://github.com/PenUniverse/PenMods-release/assets/103405023/52a6dc4e-0db3-44dd-948a-54c0bd06bb56)
这个是它的返回

**Reply by Zhuangmoumou** [reply]
好像是没有签名的数据

**Reply by Zack88604** [reply, marked as answer]
应该是没有传递参数bundleVersion
在Signature地址后添加 “?bundleVersion=1.0” 即可
试一下吧

**Reply by Zhuangmoumou** [reply]
谢谢

**Comment by Zhuangmoumou** [comment]
> 应该是没有传递参数bundleVersion 在Signature地址后添加 “?bundleVersion=1.0” 即可 试一下吧

但是现在它并不回我，有的时候还会报套接字错误

**Reply by Zack88604** [reply]
主要是看套接字错误代码是什么
常见的是代码1，大概表示chathub出错
如果返回了404页面的源代码就说明需要更换ip
